src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -450,6 +459,10 @@ public WebElement getCancelEditInstructorLink(int instrNum) {
         return browser.driver.findElement(By.id("instrCancelLink" + instrNum));
     }
 
+    public WebElement getCancelAddInstructorLink() {
+        return browser.driver.findElement(By.id("addInstrCancelLink"));
```

Comment:
The names should be consistent. As suggested, `cancelAddInstructorLink` is preferred over `addIntrCancelLink`.